src/OrchardCore/OrchardCore.Sitemaps.Abstractions/Models/CustomPathSitemapSource.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace OrchardCore.Sitemaps.Models
+{
+    /// <summary>
+    /// A sitemap source for managing custom url.
+    /// </summary>
+    public class CustomPathSitemapSource : SitemapSource
+    {
+        public static char[] InvalidCharactersForPath = ":?#[]@!$&'()*+,.;=<>\\|%".ToCharArray();
```

Comment:
Can you add `{` and `}` to this. They're also invalid characters in route templates, which we haven't got round to adding to autoroute yet